### ISSUE TITLE
Error: Activity class xxx does not exist.

### ISSUE THREAD

**Issue by nb312** [NaikSoftware/StompProtocolAndroid#100]
## when run in android studio . There is a error!
 adb shell am start -n "ua.naiksoftware.stompclientexample/ua.naiksoftware.stompclientexample.MainActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER
Error while executing: am start -n "ua.naiksoftware.stompclientexample/ua.naiksoftware.stompclientexample.MainActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER
Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=ua.naiksoftware.stompclientexample/.MainActivity }
Error type 3
Error: Activity class {ua.naiksoftware.stompclientexample/ua.naiksoftware.stompclientexample.MainActivity} does not exist.

Error while Launching activity

**Comment by forresthopkinsa** [comment]
what, are you trying to install the library itself?

**Comment by nb312** [comment]
No, I just install the example. it is usable now, but I dunno why.

**Comment by nb312** [comment]
@forresthopkinsa thank your answer!
